frontend/apps/reader/modules/readerfont.lua

```diff
@@ -82,8 +82,28 @@ function ReaderFont:init()
         })
         face_list[k] = {text = v}
     end
+    self.face_table[#self.face_table].separator = true
+    table.insert(self.face_table, {
+        text = _("Complementary standard fallback fonts"),
+        checked_func = function()
+            return G_reader_settings:nilOrTrue("complementary_fallback_fonts")
+        end,
+        callback = function()
+        G_reader_settings:flipNilOrTrue("complementary_fallback_fonts")
+            self.ui.document:setupFallbackFontFaces()
+            self.ui:handleEvent(Event:new("UpdatePos"))
+        end,
+        help_text = T(_([[
+Enable additional fallback fonts, for the most complete scripts and languages coverage.
+Complementary fonts, used in this order:
+
+%1
+
+The fallback font set with a long-press on a font name will be used before these.
+Note that the first font of the whole fallback fonts set is still used when this option is disabled.]]),
```

Comment:
Does this refer to this actual list, or the the "old" fallback font?

(In which case, it's potentially a duplicate if your global fallback is set to Noto Sans CJK SC?)

Reply:
(Not sure I understand your question :/ (the "old" fallback font?))

I tried to summarize the following with these 2 sentences :)

In https://github.com/koreader/crengine/pull/339, I explained a bit how that works:
> Users will still be able to only set one fallback font, which will be put at start of this list. If the font is already in that list, it will be moved first. This allows some bit of customisation, while always providing large coverage.

A user can set one fallback font. If not in the list: added at top. If in the list, moved to top.
So, if you set the user fallback to NotoSansCJK, this list does not change, it's the default behaviour :)
Dunno if we need to explain all this in that help_text - users shouldn't much care about duplicates.

(The only case where a duplicate/double rendering/drawing might happen is when you set as the main font one of the fonts in that list.)

Reply:
I vastly prefer that quote from the second sentence used here, FWIW.

Reply:
i.e., keep the first sentence, but replace the second with something inspired from that like `If that font happens to be part of this list, it'll be moved to the top of the list`.